.github/workflows/Tests.yml

```diff
@@ -65,11 +65,26 @@ jobs:
         working-directory: zimui
         run: |
           yarn install
+
       - name: Build
         working-directory: zimui
         run: |
           yarn build
 
+      - name: Start web server
```

Comment:
You should either move this to a dedicated job `test-zimui` or rename `build-zimui` to `build-and-test-zimui`, so that it gets a bit clearer when a job fails.

Ideally I would prefer to have a distinct `test-zimui` job so that it gets clearer what is failing when tests are failing. However since the test needs the build to be done, it is probably more realistic (for now at least) to have one single job and just rename the job for coherency.

Reply:
Renamed in [ec31f4c](https://github.com/openzim/youtube/pull/225/commits/ec31f4cddd54092bf3d6b6d75f006b249864368c)